HARK/simulation/test_monte_carlo.py

```diff
@@ -0,0 +1,113 @@
+"""
+This file implements unit tests for the Monte Carlo simulation module
+"""
+import unittest
+
+from HARK.simulation.monte_carlo import *
+
+
+pre = {
+    'R' : 1.05,
+    'aNrm' : 1,
+    'gamma' : 1.1,
+    'psi' : 1.1, # TODO: draw this from a shock,
+    'theta' : 1.1 # TODO: draw this from a shock
+}
+
+dynamics = {
+    'G' : lambda gamma, psi : gamma * psi,
+    'Rnrm' : lambda R, G : R / G,
+    'bNrm' : lambda Rnrm, aNrm : Rnrm * aNrm,
+    'mNrm' : lambda bNrm, theta : bNrm + theta,
+    'cNrm' : Control(['mNrm']),
+    'aNrm' : lambda mNrm, cNrm : mNrm - cNrm
+}
+
+dr = {
+    'cNrm' : lambda mNrm : mNrm / 2
+}
+
+
+class test_sim_one_period(unittest.TestCase):
+    def test_sim_one_period(self):
+
+        post = sim_one_period(dynamics, pre, dr)
+
+        self.assertAlmostEqual(post['cNrm'], 0.98388429)
+
+
+
+
+
+
+
+
+
+
+###############################################################3
+
+'''
```

Comment:
whoops ignore the stuff below this line. I was going to develop more robust tests from this material later.